app/assets/stylesheets/submission_similarities.css.scss

```diff
@@ -71,12 +71,31 @@ body.submission_similarities_show {
       margin-bottom: $spacer * 0.75;
     }
 
+    div.submissions-legend {
+      margin-bottom: $spacer * 0.5;
+
+      ul {
+        list-style: none;
+        margin: $spacer*0.5 0;
+        padding: 0px;
+
+        li {
+          display: inline-block;
+          padding-left: $spacer * 0.5;
+        }
+      }
+    }
+
     pre {
       margin-top: 0px;
       border: 1px solid grey;
 
       ol.linenums li.highlight {
-        background-color: rgb(255, 255, 140);
+        background-color: rgb(255, 254, 85);
```

Comment:
May want to think about how to write such values in an independent way to allow other UI enhancements (really low priority) such as dark mode.

Reply:
I didn't get it. Could you explain a bit furthermore?